src/components/00-design-tokens/_typography-mixins.scss

```diff
@@ -0,0 +1,37 @@
+@mixin antialiasing($type:antialiased) {
+  /* antialiased, none, subpixel-antialiased*/
```

Comment:
We need to add some `@if` statements here to incorporate Mozilla's different syntax for this property. (e.g. `-moz-osx-font-smoothing: grayscale;`)